assets/js/Containers/Databrowser/TimeRangeSelector.js

```diff
@@ -21,12 +23,12 @@ import {
   Tooltip
 } from "react-bootstrap";
 
-import { setTimeRange } from "./actions";
+// import { setTimeRange } from "./actions";
 import { TIME_RANGE_FILE, TIME_RANGE_FLEXIBLE, TIME_RANGE_STRICT } from "./constants";
 
 const dateRegex = /^[-]?\d{4}(-[01]\d(-[0-3]\d(T[0-2]\d(:[0-5]\d)?Z?)?)?)?$/;
```

Comment:
I know it's got nothing to do with this PR, but to cite one of my former colleagues when he reviewed on of my MR's: dark forces are at work here.

Is there a better way to do that?

Reply:
You mean the super sexy regex here? It is used only for User-Input-Validation. I guess it could be replaced by a Library (e.g. date-fns which is already part of our package) but I would rather stick to this RegEx as date-fns can convert an absurdly large amount of gibberish into valid dates which are not necessarily understood by Solr.